packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -209,21 +213,28 @@ export default class ScriptTransformer {
             // files outside `cwd` will not be instrumented
             cwd: this._config.rootDir,
             exclude: [],
+            // Needed for correct coverage as soon as we start storing a source map of the instrumented code
+            inputSourceMap: input.map,
             useInlineSourceMaps: false,
           },
         ],
       ],
+      /**
+       * Necessary to be able to map back to original source from the instrumented code
```

Comment:
Mind adding som punctuation here? 😅

Reply:
Sure, I'll do some editing for readability.